plugins/updater/src/updater.rs

```diff
@@ -94,6 +94,7 @@ pub struct UpdaterBuilder {
     endpoints: Option<Vec<Url>>,
     headers: HeaderMap,
     timeout: Option<Duration>,
+    proxy: Option<String>,
```

Comment:
We also need to provide a similar option for the JS API

Reply:
Done